proxy/grpc_handler.go

```diff
@@ -170,7 +170,6 @@ func (g GrpcProxyInterceptor) lookup(ctx context.Context, fullMethodName string)
 func (g GrpcProxyInterceptor) getDestinationHostFromMetadata(md metadata.MD) (dstHost string) {
 	dstHost = ""
 	hosts := md["dsthost"]
-	fmt.Println("hosts", md)
 	if len(hosts) == 1 {
 		dstHost = hosts[0]
 	}
```

Comment:
I know it is outside the scope of the review right now but this can be more concisely written as:
```suggestion
        kv := map[string]string{"dstHost":"www.example.com"}
	if _, ok := kv["dstHost"]; !ok {
		fmt.Print("fail")
	}
	fmt.Println(kv["dstHost"])
```

Reply:
The print "fmt.Println("hosts", md)" is only for debug purpose, it shouldn't be enabled/included in production. Otherwise each request will trigger this print, cause the performance issue.